README.adoc

```diff
@@ -3167,6 +3167,19 @@ def some_method(bar: false)
 end
 ----
 
+=== Merging Keyword Arguments [[merging-keyword-arguments]]
+
+When passing an existing hash as keyword arguments, add additional arguments directly rather than using `merge`.
```

Comment:
It'd be nice to extend this rationale a bit, as it's not clear why this is preferable, but I'll merge this as is for now.